commit 2847a572ab9f1f73a36173db1958732c61a784ff (refs/changes/06/1306/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-20T14:50:51-08:00 (2 years, 9 months ago)
    
    IMGAPI-612 IMGAPI suffering from IMGAPI-599 which breaks http_proxy support

diff --git a/package.json b/package.json
index 6670eed..7596fb1 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "progbar": "git+https://github.com/trentm/node-progbar.git#a4c56e6",
     "restify": "4.1.1",
     "rimraf": "2.2.6",
-    "sdc-clients": "10.0.1",
+    "sdc-clients": "10.0.4",
     "semver": "3.0.1",
     "sshpk": "1.10.0",
     "trace-event": "1.2.0",
